code/lib/builder-manager/src/index.ts

```diff
@@ -1,5 +1,5 @@
 import { dirname, join } from 'path';
-import { copy, writeFile, remove } from 'fs-extra';
```

Comment:
What's happening with all these changes @ndelangen? Does tsup not like named imports in some cases?

Reply:
I think what happening is that node now runs in actual ESM mode, or esbuild-register is mimicking that mode more accurately, and `fs-extra` isn';'t actually ESM.